src/main/java/frc/team2412/robot/Subsystems/ClimbMotorSubsystem.java

```diff
@@ -0,0 +1,27 @@
+package frc.team2412.robot.Subsystems;
+
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.team2412.robot.RobotMap;
+
+public class ClimbMotorSubsystem extends SubsystemBase {
+	ClimbMotorSubsystem m_ClimbMotor;
+	// private WPI_TalonSRX m_ClimbMotorSubsystem;
+
+	private WPI_TalonSRX leftMotor = RobotMap.leftClimbMotor;
+	private WPI_TalonSRX rightMotor = RobotMap.rightClimbMotor;
+
+	public ClimbMotorSubsystem(WPI_TalonSRX leftClimbMotor, WPI_TalonSRX rightClimbMotor) {
+		m_leftMotor = leftMotor;
+		m_rightMotor = rightMotor;
+
+	}
+
+	@Override
+	protected void initDefaultCommand() {
+		// TODO Auto-generated method stub
+		m_ClimbMotorSubsystem.ClimbRails();

```

Comment:
not an actual method of the subsystem

Reply:
I don't see this in my code. I'll push everything to github and if you still see it we'll talk.